File: app/src/main/java/com/channa/mobiledatausageapp/data/response/QuarterResponse.java
package com.channa.mobiledatausageapp.data.response;

public class QuarterResponse {
    private float volume_of_mobile_data;
    private String quarter;
    private int _id;

    public QuarterResponse() {
    }


    public QuarterResponse(float volume_of_mobile_data, String quarter, int _id) {
        this.volume_of_mobile_data = volume_of_mobile_data;
        this.quarter = quarter;
        this._id = _id;
    }

    public float getVolume_of_mobile_data() {
        return volume_of_mobile_data;
    }

    public void setVolume_of_mobile_data(float volume_of_mobile_data) {
        this.volume_of_mobile_data = volume_of_mobile_data;
    }

    public String getQuarter() {
        return quarter;
    }

    public void setQuarter(String quarter) {
        this.quarter = quarter;
    }

    public int get_id() {
        return _id;
    }

    public void set_id(int _id) {
        this._id = _id;
    }
}

File: app/src/main/java/com/zak/datauses/data/response/QuarterResponse.kt
package com.zak.datauses.data.response

class QuarterResponse {
    var volume_of_mobile_data = 0f
    var quarter: String? = null
    var _id = 0

    constructor() {}
    constructor(volume_of_mobile_data: Float, quarter: String?, _id: Int) {
        this.volume_of_mobile_data = volume_of_mobile_data
        this.quarter = quarter
        this._id = _id
    }

    fun getid(): Int {
        return _id
    }

    fun setid(_id: Int) {
        this._id = _id
    }
}